File: lib/src/main/java/uk/iatom/rfm69/registers/values/TestLna.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum TestLna implements IRegisterValue {
       // RegTestLna),
       NORMAL(0x1B),
       HIGH_SENSITIVITY(0x2D);

       private byte val;
       
       TestLna(int val) {
           this.val = (byte) val;
       }
   
       @Override public byte val() {
           return val;
       }
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/TestLna.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class TestLna(`val`: Int): IRegisterValue { // RegTestLna),
NORMAL(0x1B),
    HIGH_SENSITIVITY(0x2D);

    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
